src/lib/shopping-fpti-event-publisher.js

```diff
@@ -25,11 +25,12 @@ export const ShoppingEventPublisher = (config : Config) => {
    * @returns {boolean}
    */
   function isAllowedToPublishEvent() : boolean {
-    const isContainerIdSet =
-      config.propertyId !== undefined &&
-      config.propertyId !== null &&
-      config.propertyId.length > 0;
-    return isContainerIdSet;
+    // const isContainerIdSet =
+    //   config.propertyId !== undefined &&
+    //   config.propertyId !== null &&
+    //   config.propertyId.length > 0;
+    // return isContainerIdSet;
+    return true;
```

Comment:
Is it debug code?

Reply:
As mentioned in the comments, we need to disable  the check for container to control FPTI event publishing. The idea here is we will need to update TMNW Api to include fetch container by client id.